src/tests/Autosuggest.js

```diff
@@ -412,6 +426,106 @@ describe('Autosuggest', function() {
     });
   });
 
+  describe('onSuggestionFocused', function() {
+    beforeEach(function() {
+      onSuggestionFocused.mockClear();
+      onSuggestionUnfocused.mockClear();
+      createAutosuggest(
+        <Autosuggest suggestions={getSuburbStrings}
+                     onSuggestionFocused={onSuggestionFocused}
+                     onSuggestionUnfocused={onSuggestionUnfocused} />
+      );
+      setInputValue('m');
+    });
+
+    describe('Mouse interactions', function() {
+
+      it('should call onSuggestionFocused when suggestion focused using mouse', function() {
+        mouseOverFromInputToSuggestion(1);
+        expect(onSuggestionFocused).toBeCalledWith('Mordialloc');
+        expect(onSuggestionUnfocused).not.toBeCalled();
```

Comment:
We shouldn't be testing for `onSuggestionUnfocused` in the `onSuggestionFocused` section. This is the whole point of having 2 separate sections.